Pipeline/PackageBuildOutputs/PackageBuildOutputs.groovy

```diff
@@ -746,6 +746,9 @@ def parseInput(String[] cliArgs){
 		if (propertiesFile.exists()) {
 			props.packagingPropertiesFile = opts.properties
 			propertiesFile.withInputStream { props.load(it) }
+		} else {
+			println "*! [ERROR] The file '${opts.properties}' doesn't exist."
+			rc = Math.max(rc, 1)
```

Comment:
I think you can directly assign a value to `rc`, it doesn't hurt to do it this way here.
```suggestion
			rc = 2
```